static/js/publisher/release/actions/releases.js

```diff
@@ -120,18 +121,23 @@ export function releaseRevisions() {
   return (dispatch, getState) => {
     const { pendingReleases, pendingCloses, revisions, options } = getState();
     const { csrfToken, snapName, defaultTrack } = options;
-    const releases = Object.keys(pendingReleases).map(id => {
-      const pendingRelease = pendingReleases[id];
-
-      const release = {
-        id,
-        revision: pendingRelease.revision,
-        channels: pendingRelease.channels,
-        progressive: pendingRelease.progressive
-      };
-
-      return release;
-    });
+    const releases = Object.keys(pendingReleases)
+      .map(key => {
```

Comment:
Would be nice to name the `key` as `id` or `revisionId` to make it clearer what the value is.